components/accept-financial-contributions/AcceptContributionsMyself.js

```diff
@@ -0,0 +1,281 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Flex } from '@rebass/grid';
```

Comment:
Please note that `@rebass/grid` has been removed on master, this should be renamed to `../Grid`.
Sorry for the additional work!

Reply:
It's ok! I'll update it.